import-export-cli/integration/testutils/environment_testUtils.go

```diff
@@ -102,6 +102,7 @@ func ValidateExportApisPassed(t *testing.T, args *InitTestArgs, directoryName st
 	t.Cleanup(func() {
 		//Remove Exported apis
 		base.RemoveDir(directoryName + utils.TestMigrationDirectorySuffix)
+		base.WaitForIndexing()
```

Comment:
No need to wait here because we are only removing a directory and nothing related to indexing is taking place in APIM side.